.github/workflows/continuous-integration.yml

```diff
@@ -243,11 +243,81 @@ jobs:
         run: yarn build --verbose --buildtype=${{ matrix.buildtype }}
         timeout-minutes: 30
 
+      - name: Generate build details
+        run: |
+          cat > build/${{ matrix.buildtype }}/BUILD.txt << EOF
+          BUILDTYPE=${{ matrix.buildtype }}
+          BRANCH_NAME=$(echo "${GITHUB_REF#refs/heads/}")
+          RUN_ID=${{ github.run_id }}
+          REF=${{ github.sha }}
+          BUILDTIME=$(date +%s)
+          EOF
+
+      - name: Compress and archive build
+        run: tar -C build/${{ matrix.buildtype }} -cjf ${{ matrix.buildtype }}.tar.bz2 .
+
       - name: Upload build artifact
         uses: actions/upload-artifact@v2
         with:
-          name: build-${{ matrix.buildtype }}
-          path: build/${{ matrix.buildtype }}
+          name: ${{ matrix.buildtype }}.tar.bz2
+          path: ${{ matrix.buildtype }}.tar.bz2
+
+  nightwatch:
+    name: Nightwatch E2E Tests
+
+    runs-on: self-hosted
+
+    env:
+      NODE_EXTRA_CA_CERTS: /etc/ssl/certs/VA-Internal-S2-RCA1-v1.cer.pem
```

Comment:
Prefer setting this only in self-hosted runner jobs in order to reduce noise in errors in the GitHub runners about not being able to find the certs.